### PR TITLE
Update email workflow docs

### PR DESCRIPTION
**Pull request by lucaslago**
Before:
screen shot 2016-07-04 at 20 30 06

After:
screen shot 2016-07-05 at 12 03 45

I couldn't find a dev branch to point this PR to, is this [doc](https://github.com/agco-fuse/documentation/blob/master/CONTRIBUTING.md) outdated?

### CODE REVIEW

File: source/includes/external_user_workflow/_workflow.md [hunk 1]
```diff
@@ -16,16 +16,21 @@ Our team will get in contact with you as soon as posible.
...
+Once our team handle your request, you will receive two e-mails, one containing
 the following information:
 
 Property                 | Description
 --------                 | -----------
 `username`               | The provided email.
 `password`               | A random alphanumeric password generated by the system (this need to be changed).
+`URL to change password` | New users are required to change their passwords, otherwise authentication will fail.
+
+A second e-mail containing your application's identifier for [authentication](#authenticate) purposes:
```
Comment (glauco) [inline source/includes/external_user_workflow/_workflow.md]: The second one will contain

File: source/includes/external_user_workflow/_workflow.md [hunk 2]
```diff
@@ -16,16 +16,23 @@ Our team will get in contact with you as soon as posible.
 
 ## Check your e-mail inbox
 
-Once our team handle your request, you will receive an e-mail containing
+Once our team handle your request, you will receive two e-mails. The first one containing
```
Comment (andressa) [inline source/includes/external_user_workflow/_workflow.md]: s/two e-mails. The first one/two e-mails: <br/> The first one/

File: source/includes/external_user_workflow/_workflow.md [hunk 3]
```diff
@@ -16,16 +16,23 @@ Our team will get in contact with you as soon as posible.
...
 
 Property                 | Description
 --------                 | -----------
 `username`               | The provided email.
 `password`               | A random alphanumeric password generated by the system (this need to be changed).
+`URL to change password` | New users are required to change their passwords, otherwise authentication will fail.
+
+<br>
+
+The second one will contain your application's identifier for [authentication](#authenticate) purposes:
```
Comment (andressa) [inline source/includes/external_user_workflow/_workflow.md]: s/The second one will contain/The second one containing/

(just to be more likely the previous description)

File: source/includes/external_user_workflow/_workflow.md [hunk 4]
```diff
@@ -16,16 +16,24 @@ Our team will get in contact with you as soon as posible.
...
+
+The first one containing the following information:
 
 Property                 | Description
 --------                 | -----------
 `username`               | The provided email.
 `password`               | A random alphanumeric password generated by the system (this need to be changed).
+`URL to change password` | New users are required to change their passwords, otherwise authentication will fail.
+
+<br>
```
Comment (jeffhsta) [inline source/includes/external_user_workflow/_workflow.md]: Is necessary this "<br>"?

Comment (lucaslago) [inline source/includes/external_user_workflow/_workflow.md]: @jeffhsta It's supposed to put a new line between the two tables. At first I tried to add multiple empty lines in the markdown file to achieve this final behaviour, it didn't work, so I had to put this br tag :(

**Comment by glauco** [comment]
👍

**Comment by jeffhsta** [comment]
:+1:

**Pull request merged**